.github/workflows/release.yaml

```diff
@@ -0,0 +1,30 @@
+name: Release Charts
+
+on:
+  push:
+    tags:
+      - chart-*
```

Comment:
I'm assuming this controls which git tags to build. I don't think we will want to have separate tags that are only used for chart releases.

Reply:
If you plan on testing and validating the chart it needs to be released after the new Docker image has been released. As with the linting and testing above, the chart process should be seen as complimentary to the actual release and not directly part of it. For this reason I'd strongly recommend this tagging strategy; I've borrowed it from the guys over at [Flux](https://github.com/fluxcd) but I think it's a standard strategy when you have a chart directly in your repo.